packages/lib/services/interop/InteropService_Importer_Md.ts

```diff
@@ -63,6 +67,28 @@ export default class InteropService_Importer_Md extends InteropService_Importer_
 		}
 	}
 
+	// If the directory's all subdirectories are empty, we don't want to create a folder with no notes
+	private async isDirectoryEmpty(dirPath: string, stat: any) {
```

Comment:
Please give meaningful names to your variables - it's very confusing now because dirPath is not the directory you're testing, and "stat" is not the dir stats, but one of its child item stat. Maybe you need to rewrite this so that it takes only one argument - the directory to be tested.

Reply:
Done